### PR TITLE
Added testing on Python 3.12

### PR DESCRIPTION
**Pull request by jolaf**
Fix for  #290.

### CODE REVIEW

File: .github/workflows/python-package.yml [hunk 1]
```diff
@@ -12,11 +12,11 @@ on:
 jobs:
   build:
 
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```
Comment (auvipy) [inline .github/workflows/python-package.yml:15]: this do not run python 3.6

**Review by auvipy — CHANGES_REQUESTED** [review]

File: .github/workflows/python-package.yml (hunk 1 above)
Comment (jolaf) [reply .github/workflows/python-package.yml]: Oh. I see. Okay, reverted back to 20.04.

**Comment by Lawouach** [comment]
Done in 0.6.0

**Comment by auvipy** [comment]
thanks for the new release! @Lawouach

**Pull request closed without merging**